docs/src/content/docs/pt-br/reference/overrides.md

```diff
@@ -134,6 +135,12 @@ Objeto `Date` JavaScript que representa quando a página foi atualizada pela úl
 
 Objeto `URL` para o endereço onde a página poderá ser editada, se ativado.
 
+#### `labels`
+
+**Tipos:** `Record<string, string>`
+
+Um objeto content as strings da UI localizados para a página atual. Veja o guia [“Traduza a UI do Starlight”](/pt-br/guides/i18n/#traduza-a-ui-do-starlight) para uma lista de todas as chaves disponíveis.
```

Comment:
Double checking “content” is correct here and not “contendo” or “que contém”?

```suggestion
Um objeto contendo as strings da UI localizados para a página atual. Veja o guia [“Traduza a UI do Starlight”](/pt-br/guides/i18n/#traduza-a-ui-do-starlight) para uma lista de todas as chaves disponíveis.
```

Reply:
Yep, my mistake